src/libultra_code/osAiSetFrequency.c

```diff
@@ -0,0 +1,24 @@
+#include <global.h>
+#include <ultra64/hardware.h>
+
+s32 osAiSetFrequency(u32 frequency) {
+    u32 dacrate;
+    u8 bitrate;
```

Comment:
this is supposed to read as DAC Rate right? i would use camel case for both of these to make them easier to read. idk if bitrate is usually one word, but if its not maybe it should be changed as well

Reply:
I think bitrate is usually one word.